tests/unit/mixins/adapter-fetch-test.js

```diff
@@ -374,7 +374,7 @@ test('parseFetchResponseForError is able to be overwritten to mutate the error p
     }
   }).create();
 
-  const fetchReturn = this.errorAdapter.ajax({url: '/foo'});
+  const fetchReturn = this.errorAdapter.ajax({url: '/trigger-a-server-error-with-content'});
```

Comment:
You're passing in an object to `ajax`, but it should be `ajax('/trigger-a-server-error-with-content')`